btk/survey.py

```diff
@@ -74,13 +122,13 @@
     name="HST",
     pixel_scale=0.06,
     effective_area=1.0,  # TODO: placeholder
-    mirror_diameter=1.0,  # TODO: placeholder
+    mirror_diameter=2.4,  # TODO: placeholder
```

Comment:
were you going to remove all `mirror_diameter` and `effective_area` from all surveys to avoid redundancy? I still see it in a couple of surveys like here, but maybe you are in the process of editing

Reply:
I did not start removing them yet. This one is just a change because it failed the condition 0 <= `area_ratio` < 1. 
I think `mirror_diameter=1.0` and `effective_area=1.0` were just default values with 1.0 and 1.0 ? Now I think `mirror_diameter` is right but I am not sure for `effective_area`.

Reply:
yeah I don't have the numbers to check HST, so if you or someone else knows that would be good to replace them

Reply:
please specify the arguments of the method to avoid confusion
```python
define_synthetic_psf(mirror_diameter=2.4, 
                                    effective_area=1.0, 
                                    filt_wavelength=_central_wavelength["f814w"], 
                                    fwhm=0.074)
```

Reply:
Just realized I missed this one. Indeed it would be better with argument names.